src/System.Management.Automation/engine/runtime/Binding/Binders.cs

```diff
@@ -1033,6 +1033,126 @@ public override DynamicMetaObject Bind(DynamicMetaObject target, DynamicMetaObje
                     restrictions)).WriteToDebugLog(this);
             }
 
+            var iTuple = target.Value as System.Runtime.CompilerServices.ITuple;
+            if (iTuple is not null)
+            {
+                // Handle ITuple types (System.Tuple, System.ValueTuple, etc.)
+                // 3 possibilities - too few, exact, or too many elements.
+
+                var getTupleLengthExpr = Expression.Property(target.Expression.Cast(typeof(System.Runtime.CompilerServices.ITuple)), CachedReflectionInfo.ITuple_Length);
```

Comment:
There is `using` in beginning of the file.
```suggestion
                var getTupleLengthExpr = Expression.Property(target.Expression.Cast(typeof(ITuple)), CachedReflectionInfo.ITuple_Length);
```

Please remove the prefix below too.